frontend/src/components/Team.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Card, CardImg, CardBody,
+  CardTitle, CardSubtitle, Container
+} from 'reactstrap';
+import Hannah from '../images/teams-pics/Hannah.jpg';
+import Lillie from '../images/teams-pics/Lillie.jpg';
+import Jose from '../images/teams-pics/Jose.png';
+import Kirtan from '../images/teams-pics/Kirtan.png';
+import '../styles/Team.css';
+const Team = (props) => {
+
+    //Teammate constructor
+    function Teammate(name, year, major, univ, pic) {
+        this.name = name;
+        this.year = year;
+        this.major = major;
+        this.univ = univ;
+        this.pic = pic;
```

Comment:
Just a questions but why do a teammate function instead of an object?

Reply:
Since I was going to make multiple Teammate objects, I thought this would be the cleanest way to do it.